onnxruntime/python/tools/quantization/operators/activation.py

```diff
@@ -20,3 +21,49 @@ def quantize(self):
 
         quantized_value = self.quantizer.quantized_value_map[node.input[0]]
         self.quantizer.quantized_value_map[node.output[0]] = quantized_value
+
+    def quantize(self):
+        node = self.node
+        if (node.op_type == "Relu" or node.op_type == 'Clip'):
+            self.QuantizeClipRelu()
+            return
+
+        # No assert on op_type as it is controlled by registry
+        # only try to quantize when given quantization parameters for it
+        data_found, output_scale_name, output_zp_name, _, _ = self.quantizer._get_quantization_params(node.output[0])
+        if (not data_found):
+            super().quantize()
+            return
+
+        (quantized_input_names, zero_point_names, scale_names, nodes) = self.quantizer._quantize_inputs(node, [0])
+
+        qlinear_activation_output = node.output[0] + "_quantized"
+        qlinear_activation_name = ""
+        if node.name != "":
+            qlinear_activation_name = node.name + "_quant"
+        kwargs = {}
+        for attribute in node.attribute:
+            kwargs.update(_attribute_to_kwarg(attribute))
+        kwargs["domain"]=ms_domain
+
+        qlinear_activation_inputs = []
+        # Input 0
+        qlinear_activation_inputs.append(quantized_input_names[0])
+        qlinear_activation_inputs.append(scale_names[0])
+        qlinear_activation_inputs.append(zero_point_names[0])
+
+        # Output
+        qlinear_activation_inputs.append(output_scale_name)
+        qlinear_activation_inputs.append(output_zp_name)
```

Comment:
nit: why not just assign like:

qliner... = [a,b,c,d,e]

Reply:
Maybe better. changed.